### DISCUSSION TITLE
new Intel vulnerability: Snoop Assisted L1D Sampling Advisory (CVE-2020-0550 / INTEL-SA-00330)

### DISCUSSION THREAD

**Discussion by lilyanatia** [speed47/spectre-meltdown-checker#545] [Other vulnerabilities / information]
https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00330.html

https://software.intel.com/security-software-guidance/insights/deep-dive-snoop-assisted-l1-data-sampling

> A domain-bypass transient execution attack variant known as snoop-assisted L1 data sampling has been assigned CVE-2020-0550 with a CVSS of 5.6 Medium (CVSS:3.1/AV:L/AC:H/PR:L/UI:N/S:C/C:H/I:N/A:N). Under a specific set of complex conditions involving a cache-coherence snoop to a modified cache line, a malicious adversary may be able to infer the data values of some modified cache lines in the L1 data (L1D) cache using snoop-assisted L1 data sampling.

> Locally executed malicious software may be able to infer the values of data in a cache line that was modified on the same physical core by:
> 
> * Other applications
> * Operating System (OS)
> * System Management Mode (SMM)
> * Virtual Machine Monitor (VMM)

and it looks like Intel is recommending leaving operating systems vulnerable to attacks:

> Snoop-assisted L1D sampling could be mitigated by flushing the L1D cache before executing potentially malicious applications, which would require changes to the OS scheduler when hyperthreading is enabled and could impact the performance of system transitions. Because of the difficulty of this method and the performance impact caused by this mitigation, Intel does not recommend applying such mitigations to the OS.

**Comment by speed47** [comment]
Thanks for the heads up. No mitigation seems to have been included in the kernel as of now, and probably won't be, as hinted by Intel.
Keeping this open for information.

**Comment by speed47** [comment]
Documented in the FAQ https://github.com/speed47/spectre-meltdown-checker/blob/master/FAQ.md#which-rules-are-governing-the-support-of-a-cve-in-this-tool
Closing this to not declutter the issue list
